.gitignore

```diff
@@ -16,4 +16,6 @@ square/testdata
 **/*.html
 .release-env
 **/*.DS_Store
-vendor
\ No newline at end of file
+vendor
+tools/upgrademonitor/upgrademonitor
+tools/upgrademonitor/*.json
```

Comment:
added so that we don't accidentally include the upgrademonitor binary or unsigned / signed transaction JSONs in version control